hdtv/cmdline.py

```diff
@@ -459,23 +473,21 @@
         # TODO: Replace by IPython (call InteractiveShell.run_code or similar)
         self._py_console = code.InteractiveConsole(self.cmds)
 
-
         self.fPyMode = False
         self.fPyMore = False
 
         self.session = None
         self.fKeepRunning = True
         self.exit_handlers = []
 
-        if os.sep == '\\':
-            eof = 'Ctrl-Z plus Return'
+        if os.sep == "\\":
+            eof = "Ctrl-Z plus Return"
         else:
-            eof = 'Ctrl-D (i.e. EOF)'
+            eof = "Ctrl-D (i.e. EOF)"
```

Comment:
Codacy found an issue: [local variable 'eof' is assigned to but never used (F841)](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)